larky/src/main/java/com/verygood/security/larky/modules/vgs/aus/LarkyAccountUpdater.java

```diff
@@ -0,0 +1,31 @@
+package com.verygood.security.larky.modules.vgs.aus;
+
+import net.starlark.java.eval.Dict;
+import net.starlark.java.eval.EvalException;
+import net.starlark.java.eval.StarlarkInt;
+import net.starlark.java.eval.StarlarkThread;
+import net.starlark.java.eval.StarlarkValue;
+
+public interface LarkyAccountUpdater extends StarlarkValue {
+  /**
+   * Get updated info for the provided card
+   *
+   * @param number card's number
+   * @param expireMonth card's expiration month
+   * @param expireYear card's expiration year
+   * @param name the name on the card
+   * @param clientId client id of service account to access calm API
+   * @param clientSecret client secret of service account to access calm API
+   * @param thread Starlark thread object
+   * @return a dict contains the network token values
+   */
+  Object lookupCard(
+      String number,
+      StarlarkInt expireMonth,
+      StarlarkInt expireYear,
+      String name,
+      String clientId,
+      String clientSecret,
```

Comment:
secret? 🤔

Reply:
please see my comment at https://github.com/verygoodsecurity/starlarky/pull/442#discussion_r1463854857 explaining the rationale